src/commands/app/deploy.ts

```diff
@@ -54,7 +49,7 @@ async function archiveMicroserviceSrc(
     ) => Promise<string[]>;
   } = {
     '.dockerignore': dockerignorePaths,
-    '.skyignore': skyignorePaths
+    '.skyignore': dockerignorePaths
```

Comment:
Since we always use `dockerignorePaths` now, `ignoreFuncMap` is now redundant.

Reply:
`ignoreFuncMap` was removed in commit `0770f51`